packages/tupaia-web/src/api/queries/useReport.ts

```diff
@@ -16,24 +16,36 @@ export const useReport = (
   reportCode?: DashboardItemType['reportCode'],
   itemCode?: DashboardItemType['code'],
   legacy?: DashboardItemType['legacy'],
+  startDate?: string | null,
```

Comment:
this is quite a long list of function parameters. You could update it to be useReport(reportCode, params) where params is an object but I'm not sure